src/Components/ScrivitoExtensions/SocialCardsTab.js

```diff
@@ -3,7 +3,7 @@ import * as Scrivito from "scrivito";
 import getMetadata from "../../utils/getMetadata";
 
 Scrivito.registerComponent("SocialCardsTab", ({ obj }) => (
-  <div className="">
+  <div className={`scrivito_${Scrivito.uiContext()?.theme ?? "light"}`}>
```

Comment:
* [ ] Falling back to the light theme while `uiContext` is `null` will cause a color flash. We should not render while the UI context is loading.

Reply:
See https://github.com/Scrivito/scrivito_example_app_js/pull/428#discussion_r742639618